http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -476,6 +476,7 @@ class ClusterMonitorActor(
     runtimeAndRuntimeConfig: RuntimeAndRuntimeConfig
   )(implicit ev: ApplicativeAsk[IO, TraceId]): IO[ClusterMonitorMessage] =
     for {
+      // TODO: maybe this can be replaced with `.getRuntimeStatus` on `RuntimeAlgebra`
```

Comment:
Sounds reasonable

Reply:
I see you made a similar change in `ZombieRuntimeMonitor` - would be good to do here as well